support-frontend/assets/pages/weekly-subscription-checkout/components/thankYou.jsx

```diff
@@ -126,12 +126,11 @@ function ThankYouContent({
     ] :
     [
       <span>
-        Look out for an email from us confirming your subscription.
-        It has everything you need to know about how to manage it in the future.
+        Look out for an email from us confirming your subscription. It will contain everything you need to know.
       </span>,
       <span>
-        Your magazine will be delivered to your door.{' '}
-        <a className="thank-you-link" href={homeDeliveryUrl}>Here&apos;s a reminder of how home delivery works</a>.
+        Your magazine will be delivered to your door.
+        Please allow 1-7 days after publication date for your magazine to arrive, depending on national post services.
```

Comment:
I would suggest changing the text here to '1 to 7 days' as this will be clearer when read out by a screen reader.